spec/controllers/api/v1/pets_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::PetsController, type: :controller do
+  render_views
+
+  describe 'when map (:top param) is present' do
+    it 'responds with 200 code' do
+      get :index, { :format => :json, :top => '49' }
```

Comment:
Jeff, i would change this -> { :format => :json, :top => '49' } to -> {format: :json, top: '49'}